.travis.yml

```diff
@@ -56,6 +56,13 @@ before_cache:
   - brew cleanup
 
 before_install:
+  - docker pull bitnami/zookeeper:3.6.0
```

Comment:
Not all CI environments support docker, such as OSX.

So we should complete the installation in a specific system script.

Here is an example: https://github.com/apache/incubator-apisix/blob/master/.travis/linux_openresty_runner.sh#L35

Reply:
Sure.Since all tests passed which means we are currently skipping testcases run on osx?

Reply:
After this PR was merged, we started running test cases for the OSX system in each PR.

https://github.com/apache/incubator-apisix/pull/1314/files

Reply:
@membphis 

I hope in the future we will be not running unit tests on 

OSNAME=linux_apisix_master_luarocks
OSNAME=linux_apisix_current_luarocks

Which I dont see currently we are running unit tests

Reply:
@membphis  updated